app/controllers/sign_in/user_info_controller.rb

```diff
@@ -6,28 +6,13 @@ class UserInfoController < ApplicationController
 
     def show
       authorize access_token, policy_class: SignIn::UserInfoPolicy
+      user_info = SignIn::UserInfo.from_user(current_user)
 
-      render json: user_info_json, status: :ok
+      render json: user_info.serializable_hash.compact, status: :ok
```

Comment:
I'm not sure we want to compact, let's just send everything to them even if it's `nil` for now